data/sql/updates/pending_db_characters/rev_1621780938723425400.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_characters` (`sql_rev`) VALUES ('1621780938723425400');
+
+ALTER TABLE `mail` 
+ADD COLUMN `auctionId` INT(11) DEFAULT 0 NOT NULL AFTER `checked`; 
```

Comment:
do not put the length to the INT type, it will be deprecated soon in the next mysql version

Reply:
good catch